org.viking.Viking.yml

```diff
@@ -0,0 +1,183 @@
+app-id: org.viking.Viking
+# https://docs.flatpak.org/en/latest/available-runtimes.html
+# NB Viking doesn't use any Gnome specifics so freedesktop should suffice for the build
+# Albeit Yelp for help is Gnome only
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: viking
+rename-icon: viking
+rename-desktop-file: viking.desktop
```

Comment:
Ideally these renames should happen upstream.

Reply:
?
I presumed the intention of these renames were such that upstream didn't need any changes (or create otherwise duplicate files) so the flatpak build would rename the existing icon / desktop files that exist from 'program.extension' --> 'org.program.Program.extension'